views.py

```diff
@@ -0,0 +1,98 @@
+from flask import flash, redirect, render_template, request, url_for
+from flask_login import current_user, login_user, logout_user, login_required
+from __init__ import app, db, login_manager, bcrypt
+import forms
+from models import User, Group, Bill
+
+
+@login_manager.user_loader
+def load_user(user_id):
+    return User.query.get(int(user_id))
+
+
+@app.route("/register", methods=['GET', 'POST'])
+def register():
+    db.create_all()
+    if current_user.is_authenticated:
+        return redirect(url_for('login'))
+    form = forms.RegistrationForm()
+    if form.validate_on_submit():
+        encrypted_password = bcrypt.generate_password_hash(
```

Comment:
Pamiršai pridėti patikrinimą ar dar tokio vartotojo email nėra (mačiau, kad formos klasėj yra toks metodas).

```suggestion
        form.check_email(email=form.email)
        encrypted_password = bcrypt.generate_password_hash(
```